main.go

```diff
@@ -375,6 +385,8 @@ func reindexDeploymentsReporting(
 		if len(dd) < deviceDeploymentsBatchSize {
 			break
 		}
+		nextIterAt.Add(throttleDuration)
+		time.Sleep(time.Until(nextIterAt))
```

Comment:
I understand why you are doing it like that, but to be mean: what in case of the time change?

Reply:
Right, I should use a timer to ensure that it's using monotonic time.